src/neo/Network/P2P/Payloads/ExtensiblePayload.cs

```diff
@@ -90,7 +90,7 @@ void IVerifiable.SerializeUnsigned(BinaryWriter writer)
             writer.WriteVarBytes(Data);
         }
 
-        internal bool Verify(ProtocolSettings settings, DataCache snapshot, ISet<UInt160> extensibleWitnessWhiteList)
+        public bool Verify(ProtocolSettings settings, DataCache snapshot, ISet<UInt160> extensibleWitnessWhiteList)
```

Comment:
If this is `public`, is it still necessary to expose `UpdateExtensibleWitnessWhiteList`?

Reply:
`UpdateExtensibleWitnessWhiteList` could be moved to `ExtensiblePayload` as a Helper for return the default values.

Reply:
I think it's enough to expose `Verify` only.